common/src/Model/Polyhedron_ConvexHull.h

```diff
@@ -36,6 +37,27 @@
 
 namespace TrenchBroom {
     namespace Model {
+        namespace {
+            template <typename T, typename FP, typename VP>
+            vm::plane<T,3> makePlaneFromBoundary(const Polyhedron_HalfEdgeList<T,FP,VP>& boundary) {
+                if (boundary.size() < 3) {
+                    throw GeometryException("boundary must have at least thee vertices");
+                }
+                
+                const auto* first = boundary.front();
+                const auto& p1 = first->next()->origin()->position();
+                const auto& p2 = first->origin()->position();
+                const auto& p3 = first->previous()->origin()->position();
+                
+                const auto [valid, plane] = vm::from_points(p1, p2, p3);
```

Comment:
Bad function name.. should be `vm::plane_from_points` (vecmath issue I guess)

Reply:
Created an issue: https://github.com/kduske/vecmath/issues/18

Reply:
Changed in 742d8f96cc61d0454a15aa62cc444aab378c194b.